src/Compilers/Core/Portable/WellKnownMembers.cs

```diff
@@ -2858,10 +2858,11 @@ static WellKnownMembers()
                 (byte)(MemberFlags.Method | MemberFlags.Static),                                                                                    // Flags
                 (byte)WellKnownType.ExtSentinel, (byte)(WellKnownType.Microsoft_CodeAnalysis_Runtime_Instrumentation - WellKnownType.ExtSentinel),  // DeclaringTypeId
                 0,                                                                                                                                  // Arity
-                    4,                                                                                                                              // Method Signature
+                    5,                                                                                                                              // Method Signature
```

Comment:
We will make some effort to enable a given version of the runtime to work with either newer or older compilers, but there might be reductions in capability when there are mismatches.